### ISSUE TITLE
Update ?

### ISSUE THREAD

**Issue by mikhoul** [Gitoffthelawn/uBlock-Deluxe#7]
Do you have an update ?  :question: 

Regards :smile:

**Comment by Gitoffthelawn** [comment]
Yes, I have significant updates for anyone interested.  What blocker are you using, and what version?

**Comment by mikhoul** [comment]
Thanks for the quick reply,

I use uBlock Origin v1.9 beta since last week, I was using ABP before but UB Origin have more advanced blocking features BUT a big drawback is the unfriendly and unintuitive interface....

I miss a lot ABP friendy GUI and functions, UBO GUI is made  almost only for programmer ,  one of the worst interface. I've seen in longtime :crying_cat_face: 

That's why I was looking for some "skin" or addon to make it more friendly and I found this repository. 

Let me know where I can find your latest update please !   :octocat: 

Regards ! :+1:

**Comment by Gitoffthelawn** [comment]
You're welcome.

Yes, I agree completely about the UI/UX issues surrounding uBlock and uBlock Origin.

uBlock Deluxe features scores of improvements, but since it is an overlay, it cannot fix many of the underlying issues.  So, in my opinion, it's several steps in the right direction... but don't expect more than a CSS overlay can offer! :-)

My time is currently overbooked, but I've put your request for the latest update onto my projects list.  I'll post here as soon as I upload the latest update.

If you get desperate, and I'm not fast enough for your needs, don't hesitate to ping me in this thread.  It won't bother me, and I'll see what I can do to reprioritize some tasks so I can help you out.

**Comment by mikhoul** [comment]
I will gladly appreciate any CSS that will relieve my pain :smiley: 

You can send me just the plain CSS files I will integrate it with Stylish that I use to manage my CSS style.

Maybe later in my spare time I will do an Userscript to help to manage and add features to the "local pages" of UBO like the logger.

Best Regards and wishing you a great day ! :+1:

**Comment by Gitoffthelawn** [comment]
Thanks!  You as well! :+1: 

The latest uBlock Deluxe even improves the uBlock logger.  It makes enhancements to every section of the uBlock UI.

Since you are using Stylish, I might upload a version specifically for that extension... I might even be able to figure out a way so Stylish will automatically update the CSS with changes I make here on GitHub.  I think it will be rather easy.  I'll take a look at it when I have some time.

More importantly, I'll work towards getting you the latest version.

Best regards to you! :smiley:

**Comment by mikhoul** [comment]
Hey it's me again,  :hand: 

I see that you have also done another CSS hack for the REDIRECTOR add-on would you mind to include it with the latest version of uBlock Deluxe ? 

BTW: I asked to Gorhill for a nice change with Ublock that you should like https://github.com/gorhill/uBlock/commit/269c35a04a668927d7fe917ba5f175efc92c4db2

You will be able to open the logger directly in his own window: The first time you will have to press the _shift key_ and the logger button so it will open in a new windows instead of a tab and it will remember the next time that you want it directly (without pressing "_shift_") in a new window (not a tab).

For any reason you want it back to open in a tab just press again the shift key and the logger button at the same time.  😊

I will wait a few weeks and will find something  else to ask....  One feature request at time :stuck_out_tongue_winking_eye:

Have a nice day ! ✌️

**Comment by ddraws** [comment]
Would be awesome if you guys posted some screenshots.

**Comment by mikhoul** [comment]
@ddraws  I wait for the reply of @Gitoffthelawn  since the end of September to have the latest CSS files.

Regards :octocat:

**Comment by Gitoffthelawn** [comment]
@mikhoul My ears must have been burning.  I was just working on an update for the latest version of uBlock Origin! 👍

**Comment by mikhoul** [comment]
Thanks for the reply !    😋

**Comment by Gitoffthelawn** [comment]
@mikhoul @ddraws  New release of uBlock Deluxe is available!

**Comment by mikhoul** [comment]
:+1:    

Thanks !  :smiley:

**Comment by Gitoffthelawn** [comment]
@mikhoul You're welcome!  Enjoy!

**Comment by Atavic** [comment]
@Gitoffthelawn you've been invited to https://github.com/ghacksuserjs a newly created organization on github.

**Comment by Gitoffthelawn** [comment]
@Atavic Thank you.  I have responded here: https://github.com/ghacksuserjs/ghacks-user.js/issues/1#issuecomment-281043822
